### PR TITLE
Dynamic theme fixes for theinformation.com

### PR DESCRIPTION
**Pull request by michaelkotrous**
Invert the logo and tweak styles of the share button

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -5097,6 +5097,20 @@ INVERT
 
 ================================
 
+theinformation.com
+
+INVERT
+.logo
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:5103]: Seems to me it's already inverted without this fix

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -5097,6 +5097,20 @@ INVERT
...
+
+CSS
+article.single .sticky-header button.button-share,
+#bookmark-and-share button.button-share {
+    background-color: #21498c;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:5108]: I assume this are the default values from the site itself?

The articles is after a paywall so I can't check for myself.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (michaelkotrous) [reply src/config/dynamic-theme-fixes.config:5103]: Good catch. I only looked at the article pages on the other side of the paywall. It looks like the gated homepage is the only page with a logo that inverts correctly. Adding some CSS will fix the box-shadows on the gated homepage too. Pasting my new devtools code below.

```suggestion
theinformation.com

INVERT
.logo

CSS
article.single .sticky-header button.button-share,
#bookmark-and-share button.button-share {
    background-color: #21498c;
    color: #e8e6e3;
}
article.single .sticky-header button.button-share:hover,
#bookmark-and-share button.button-share:hover {
    background-color: #245098;
}
.locked section .insert-container {
    -webkit-filter: drop-shadow(20px 20px 20px #26292b);
    filter: drop-shadow(20px 20px 20px #26292b);
}
.locked section .insert-container#app-screenshot-container {
    -webkit-filter: drop-shadow(20px -20px 20px #26292b);
    filter: drop-shadow(20px -20px 20px #26292b);
}

IGNORE IMAGE ANALYSIS
header.locked .logo

================================

```

Screenshot of article page with this code applied.

Screen Shot 2020-08-17 at 11 46 03 AM

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (michaelkotrous) [reply src/config/dynamic-theme-fixes.config:5108]: These color values match other buttons on the site. You can confirm with the blue button on the [log-in page](https://www.theinformation.com/sessions/new).

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:5103]: You can drop the `-webkit-filter` It's not needed for fixes for the rest it LGTM make sure check for `light` scheme in dark reader if it don't break the site.

Comment (michaelkotrous) [reply src/config/dynamic-theme-fixes.config:5103]: Thanks. Double-checked the light theme and committed changes with `-webkit-filter` dropped.

**Pull request merged**